components/binding-usage-controller/docs/proposals/usage-kind.md

```diff
@@ -86,12 +86,106 @@ spec:
     kind: function
     name: redis-client
 ```
-The ServiceBindingUsage **spec.usedBy.kind** field matches the name of the **UsageKind** instance.
+The ServiceBindingUsage **spec.usedBy.kind** field matches the name of the UsageKind instance.
 
 ## Testing
 
-It is important to have an easy way to create a test for a new **UsageKind**. There should be a way to check that the proper RBAC role is set or that the **labelsPath** field is correct. The purpose is to design a solution that makes writing such acceptance tests as easy as possible.
+It is important to have an easy way to create a test for a new UsageKind. There should be a way to check that the proper RBAC role is set or that the **labelsPath** field is correct. The purpose is to design a solution that makes writing such acceptance tests as easy as possible.
+
+## UsageKind and ServiceBindingUsage finalizers
```

Comment:
IMO we have a missing part here.
First. why we need this at all and second, what are the problems with using finalizers vs storing data in cfg map